model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -539,7 +547,7 @@ def _calculate_avg_first_token_latency(self, llm_output: Dict) -> float:
                 request["response_timestamps"][0] - request["timestamp"]
             )
 
-        avg_first_token_latency = mean(total_first_token_latencies)
+        avg_first_token_latency = float(mean(total_first_token_latencies))
```

Comment:
We can use `np.mean`. I think it returns float as default.

Reply:
I am using `np.mean` it returns a `float64`, but we're expecting `float` so I'm converting it.